prow/config/config.go

```diff
@@ -75,6 +76,18 @@ type Trigger struct {
 	TrustedOrg string `json:"trusted_org,omitempty"`
 }
 
+// SlackEvent is config for the slackevents plugin.
+// If a PR is pushed to any of the repos listed in the config
+// then sent message to the all the  slack channels listed if pusher is NOT in the whitelist.
+type SlackEvent struct {
+	// Repos is either of the form org/repos or just org.
+	Repos []string `json:"repos,omitempty"`
+	// List of channels on which a event is published.
+	Channels []string `json:"channels,omitempty"`
+	// A slack event is published if the user is not part of the WhiteList.
+	WhiteList []string `json:"whitelist,{\"k8s-merge-robot\"}"`
```

Comment:
Will this notation default k8s-merge-robot? Can we separate defaulting from the schema?